### PR TITLE
[BLOG] Add PyCon US 2023 - An action-packed week

### PR DESCRIPTION
**Pull request by lysnikolaou**
## Text styling

- [X] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [X] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [X] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [X] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [X] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [X] If there are emojis, there are not more than three in a row.
- [X] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [X] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/pycon-us-2023-an-action-packed-week.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+title: 'PyCon US 2023 - An action-packed week'
+published: June 28, 2023
+author: nikolaou-lysandros
```
Comment (bskinn) [inline apps/labs/posts/pycon-us-2023-an-action-packed-week.md:4]: Your slug is in Storyblok as `firstname-lastname`

```suggestion
author: lysandros-nikolaou
```

**Comment by kgryte** [comment]
@rgommers Would you mind reviewing this blog post? :)

File: apps/labs/posts/pycon-us-2023-an-action-packed-week.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+posts on the Language Summit](https://pyfound.blogspot.com/2023/05/the-python-language-summit-2023_29.html).
+He really did an amazing job in summing up all of the talks and discussions. I will,
+however, mention one personal highlight from the Language Summit. Before the first
+talk, Pablo Galindo Salgado, Python 3.10 & 3.11 release manager and fellow compiler
+front-end co-conspirator, went up to the podium and, in front of everyone, merged
+[PR #102855](https://github.com/python/cpython/pull/102855), the implementation of
+PEP 701, which marked a significant milestone in our work toward standardizing and
+improving f-strings.
+
+The next few days passed quickly by. On Thursday I attended the opening reception.
```
Comment (rgommers) [inline apps/labs/posts/pycon-us-2023-an-action-packed-week.md:36]: minor textual: "passed by quickly"

File: apps/labs/posts/pycon-us-2023-an-action-packed-week.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+It was the first time we got to see the expo hall with all the different companies
+and sponsors of PyCon US, which also included our very own [OpenTeams](https://www.openteams.com/).
```
Comment (rgommers) [inline apps/labs/posts/pycon-us-2023-an-action-packed-week.md:38]: "our very own OpenTeams" -> Quansight and our sister company OpenTeams

File: apps/labs/posts/pycon-us-2023-an-action-packed-week.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+some people were going to stay around for the sprints the following week, but the
+sound of all the people rushing to their talks, talking with each other and having
+fun is not the same. The day started with a round of lightning talks and an
+eye-opening keynote by Margaret Mitchell on data, bias, and all the things we should
+be watching out for in the AI era, and it ended with three truly special keynotes.
+In the first one, Carol Willing talked about Python’s global network and how there are
+three basic elements to it: connection, communication, and scale. The second one was
+Deb Nicholson’s update on the PSF and the giving of Community Service Awards. The
+last talk of the day, which also marked the end of the 20th PyCon US, was a trip down
+memory lane by Guido van Rossum, who told us stories about the first python conferences,
```
Comment (rgommers) [inline apps/labs/posts/pycon-us-2023-an-action-packed-week.md:75]: typo: python needs capitalization

**Review by rgommers — APPROVED** [review]
This reads really well, thanks @lysnikolaou! Only a few minor comments from me. And apologies for the delay - I had somehow missed this PR being opened.

**Comment by lysnikolaou** [comment]
Thanks a lot for the review @rgommers! Feedback addressed.

**Pull request merged**